### DISCUSSION TITLE
Default tab index templatable

### DISCUSSION THREAD

**Discussion by Rudd-O** [kinghat/tabbed-card#70] [Feature Requests]
Hi there!  I want to use the tabbed card to select amongst multiple media players.  The default tab index should be templatable so, upon first render, HA elects to show me the media player that is, in fact, currently active, or at least on.

Is that possible?  Thanks.

**Comment by kinghat** [comment]
im not sure but ive converted to a feature request in the meantime.

**Reply by Rudd-O** [reply]
Thank you.  I think this deserves discussion indeed.

**Comment by spasma** [comment]
@Rudd-O I've added this feature in PR: https://github.com/kinghat/tabbed-card/pull/105

Please try it out.

**Comment by Rudd-O** [comment]
Wow mang, you are awesome.  When's release?  That way I can give it a shot.

**Comment by ilearn2023au** [comment]
any updates?

**Comment by msolomos** [comment]
this feature is really great and I believe that it will establish greater acceptance of the tabbed card. 
For example, on a tablet's dashboard show active entity  (central heat at winter / A/C at summer)

Basically, even though it is not possible with template, an option to show default tab based on state would be fine
